core/src/test/java/io/questdb/test/griffin/engine/functions/date/TimestampAddFunctionFactoryTest.java

```diff
@@ -341,11 +361,31 @@ public void testSecondSimple() throws SqlException {
         call('s', 5, 1587275359886758L).andAssert(1587275364886758L, 0.0001);
     }
 
+    @Test
+    public void testMilliSimple() throws SqlException {
+        call('T', 5, 1587275359886758L).andAssert(1587275359891758L, 0.0001);
+    }
+
+    @Test
+    public void testMicroSimple() throws SqlException {
+        call('T', 5, 1587275359886758L).andAssert(1587275359891758L, 0.0001);
```

Comment:
Test name suggests that it should use `'u'`:
```suggestion
        call('u', 5, 1587275359886758L).andAssert(1587275359891758L, 0.0001);
```

Reply:
You are right! Sorry! Too much copy and paste. Will change

Reply:
fixed